src/js/utils/general.js

```diff
@@ -99,12 +99,19 @@ export function getPopperOptions(attachToOptions, step) {
         options: {
           altAxis: true
         }
+      },
+      {
+        name: 'focusAfterRender',
+        enabled: true,
+        phase: 'afterWrite',
+        fn() {
+          setTimeout(() => {
+            step.el.focus();
```

Comment:
@chuckcarpenter this modifier is required if we want to show/hide the popper when scrolling away. We have to focus every time the popper recalculates basically, so if the element is shown it gets focused.